.circleci/config.yml

```diff
@@ -28,12 +28,14 @@ jobs:
       - restore_cache:
           keys:
             - v1-mix-cache-{{ checksum "mix.lock" }}
+      - run: mix hex.organization auth smartrent --key $SMARTRENT_HEX_KEY
       - run: mix deps.get
       - run: mix compile --warnings-as-errors
       - run: mix test
       - run: mix docs
       - run: mix format --check-formatted
       - run: mix deps.unlock --check-unused
+      - run: mix credo --ignore-checks "refactor"
```

Comment:
I assume that there aren't too many refactor tasks since it's a newer repo. So I'd change this to be plain mix credo or even mix credo --strict if the current issues are easy to swap

Reply:
I'll fix the credo warnings in other PRs